components/Modals/CreateItemModal.tsx

```diff
@@ -372,6 +432,116 @@ export const CreateItemModal: React.FC<CreateItemModalProps> = ({
+              <Image
+                alt="down arrow"
+                height="14px"
+                mb={2}
+                src="/icons/question-mark.svg"
+                width="14px"
+              />
+            </Tooltip>
+          </Flex>
+          <Switch
+            isChecked={classRequirementsToggle}
+            onChange={e => setClassRequirementsToggle(e.target.checked)}
+          />
+        </FormControl>
+
+        {classRequirementsToggle && (
+          <SimpleGrid columns={4} spacing={3} w="100%">
+            {game?.classes.map(c => (
+              <Button
+                h="200px"
+                key={c.id}
+                onClick={() => {
+                  if (classRequirements.includes(c.classId)) {
+                    setClassRequirements(
+                      classRequirements.filter(cr => cr !== c.classId),
+                    );
+                  } else {
+                    setClassRequirements([...classRequirements, c.classId]);
+                  }
+                }}
+                variant="unstyled"
+                width="100%"
+              >
+                <VStack
+                  background={
+                    classRequirements.includes(c.classId) ? 'black' : 'white'
+                  }
+                  border="3px solid black"
+                  borderBottom="5px solid black"
+                  borderRight="5px solid black"
+                  color={
+                    classRequirements.includes(c.classId) ? 'white' : 'black'
+                  }
+                  cursor="pointer"
+                  fontWeight={600}
+                  h="100%"
+                  justify="space-between"
+                  px={5}
+                  py={3}
+                >
+                  <Image
+                    alt={`${c.name} image`}
+                    h="70%"
+                    objectFit="contain"
+                    src={c.image}
+                    w="100%"
+                  />
+                  <Text textAlign="center" fontSize="14px">
+                    {c.name}
+                  </Text>
+                </VStack>
+              </Button>
+            ))}
+          </SimpleGrid>
+        )}
+        <FormControl isInvalid={showError && !itemSupply}>
+          <Flex align="center">
+            <FormLabel>Is this item soulbound?</FormLabel>
+            <Tooltip label="By making this item soulbound, you prevent characters who hold the item from ever being able to transfer it.">
+              <Image
+                alt="down arrow"
+                height="14px"
+                mb={2}
+                src="/icons/question-mark.svg"
+                width="14px"
+              />
+            </Tooltip>
+          </Flex>
+          <Switch
+            isChecked={soulboundToggle}
+            onChange={e => setSoulboundToggle(e.target.checked)}
+          />
+        </FormControl>
+        <FormControl isInvalid={showError && !itemSupply}>
```

Comment:
Do you want to comment this out for this PR ?

Reply:
Yeah, makes sense to!

Reply:
Commented out in 8e1b26ce4aaf6dfc6d82147f1f9698a9701a7175